.github/workflows/ci.yaml

```diff
@@ -0,0 +1,123 @@
+# File generated by bin/generate-actions-yaml.sh
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-images:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push - codercom/enterprise-configure:ubuntu
+        uses: docker/build-push-action@v2
+        with:
+          context: ./images/configure
+          file: ./images/configure/Dockerfile.ubuntu
+          tags: codercom/enterprise-configure:ubuntu
+          push: ${{ github.event_name != 'pull_request' }}
+      -
+        name: Build and push - codercom/enterprise-goland:centos
+        uses: docker/build-push-action@v2
+        with:
+          context: ./images/goland
+          file: ./images/goland/Dockerfile.centos
+          tags: codercom/enterprise-goland:centos
```

Comment:
Currently it doesn't. `tags` does allow for multiple tags, so we could add that to the ubuntu ones. But what's the value of the latest tag? Most things I read up suggest not using it because it's unintuitive [1][2][3]

1. https://vsupalov.com/docker-latest-tag/
2. https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375
3. https://blog.container-solutions.com/docker-latest-confusion